docs/use-cases/manage-modules.md

```diff
@@ -1,277 +1,37 @@
 ---
-title: "Upload your Own Modules to the Viam Registry"
-linkTitle: "Upload your module"
-weight: 20
+title: "How to manage modules you created"
+linkTitle: "Manage modules"
 type: "docs"
-tags:
-  [
-    "server",
-    "rdk",
-    "extending viam",
-    "modular resources",
-    "components",
-    "services",
-  ]
-description: "Use the Viam CLI to upload a custom module to the Viam registry as a public module or as a private module that is shared only within your organization."
-no_list: true
-icon: true
-images: ["/registry/upload-module.svg"]
+weight: 26
+images: ["/registry/create-module.svg"]
+tags: ["modular resources", "components", "services", "registry"]
+description: "Update or delete your existing modules, or change their privacy settings."
 aliases:
-  - "/extend/modular-resources/upload/"
-  - "/modular-resources/upload/"
+  - /use-cases/deploy-code/
 ---
 
-Once you have [created a custom module](/registry/create/), use the instructions on this page to upload it to the Viam registry as a public module that is shared with other Viam users, or as a private module that is shared only within your [organization](/cloud/organizations/).
+After you [create and deploy a module](/use-cases/create-module/), you may need to update or delete it.
```

Comment:
What if it's called "manage or delete"? The delete stuff should be pretty trivial, i.e. intuitive in the app, so I barely think we need it in the docs so IDK if it deserves its own page?

I think it's related to this page because if someone say, changes their module from private to public, that may change their ability to delete it later.

Reply:
possibly only a mention in the reference section of the docs then. Let's keep it as is for now and iterate later